commercial/app/controllers/commercial/HostedContentController.scala

```diff
@@ -4,13 +4,13 @@ import common.commercial.HostedPage
 import model.Cached.RevalidatableResult
 import model.{Cached, NoCache}
 import play.api.mvc.{Action, Controller}
-import views.html.hosted.guardianHostedPage
+import views.html.hosted.{guardianHostedPage, guardianHostedVideo}
 
 object HostedContentController extends Controller {
 
   def renderHostedPage(pageName: String) = Action { implicit request =>
     HostedPage.fromPageName(pageName) match {
-      case Some(page) => Cached(60)(RevalidatableResult.Ok(guardianHostedPage(page)))
+      case Some(page) => Cached(60)(RevalidatableResult.Ok(guardianHostedPage(page, guardianHostedVideo(page))))
```

Comment:
Could call `guardianHostedPage` template from inside `guardianHostedVideo` template, then this becomes:
`RevalidatableResult.Ok(guardianHostedVideo(page))`